social.install

```diff
@@ -74,3 +74,18 @@ function _social_core_disable_secondary_gin_toolbar() {
 function social_update_11501() {
   _social_core_disable_secondary_gin_toolbar();
 }
+
+/**
+ * Enable author info on book pages.
```

Comment:
@nkoporec I see we moved book-related config files here: https://github.com/goalgorilla/open_social/pull/1744

But anyway I think it makes more sense to add this update hook in the `social_book` module. Then we can also remove check if `moduleExists`.